code/modules/surgery/organs/external/_external_organs.dm

```diff
@@ -250,13 +255,18 @@
 				return
 			var/mob/living/carbon/human/human_owner = ownerlimb.owner
 			draw_color = human_owner.hair_color
-	color = draw_color
+	if(!apply_color_to_layer)
+		color = draw_color
```

Comment:
couldn't we just make this 
```suggestion
	color = get_overlay_color(image_layer)
```

Reply:
The color here is the color that applies to the whole object. The actual overlays are generated in generate_and_fetch_overlays based on a bunch of mob characteristics and can’t be changed from here. Inherit colors isn’t called very frequently and is just to set the initial color to draw with